### PR TITLE
fix: add Anu Wat

### PR DESCRIPTION
**Pull request by anuwatavis**

### CODE REVIEW

**Review by emmalearnscode — APPROVED** [review]

File: public/data/taravis.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Anu Wat",
+  "bio": "Web Dev, Interactive Media Dev",
```
Comment (Pradumnasaraf) [inline public/data/taravis.json:3]:
```suggestion
  "bio": "Web Dev | Interactive Media Dev",
```

Comment (Pradumnasaraf) [inline public/data/taravis.json:3]:
```suggestion
  "bio": "Web Dev | Interactive Media Dev",
```

File: public/data/taravis.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "avatar": "https://github.com/anuwatavis.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/anuwatavis.png",
```
Comment (Pradumnasaraf) [inline public/data/taravis.json:8]:
```suggestion
      "url": "https://github.com/anuwatavis",
```
By mistake you have add avatar link

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM
Thank you for contributing to `LinkFree`

**Review by krishguptadev — APPROVED** [review]

File: public/data/taravis.json (hunk 1 above)
Comment (krishguptadev) [reply public/data/taravis.json]: I think the content should be up to them unless it is a grammar/spelling mistake 🙂

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**